Testing/SystemTests/tests/analysis/ILLIndirectReductionFWS.py

```diff
@@ -114,3 +115,13 @@ def _run_efws_mirror_sense(self):
         avg = numpy.average(yData)
         for y in numpy.nditer(yData):
             self.assertDelta(y, avg, 0.001)
+
+    def _run_calib_bg(self):
+        # this tests with the background file for calibration
+        IndirectILLReductionFWS(Run="143720:143728:2",
+                                CalibrationRun="143721",
+                                CalibrationBackgroundRun="143723",
+                                CalibrationBackgroundScalingFactor=0.1,
+                                OutputWorkspace="efws_calib_bg")
+
+        self.assertEquals(mtd['efws_calib_bg_red'].getItem(0).getNumberHistograms(), 18)
```

Comment:
Can we assert some Y data values so future developers are sure their changes don't affect the results.